README.md

```diff
@@ -11,3 +9,59 @@ Make sure that you have the compatible versions of Visual Studio with .NET MAUI
etplace), [ASP.NET MVC](https://www.syncfusion.com/aspnet-mvc-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [ASP.NET Web Forms](https://www.syncfusion.com/jquery/aspnet-webforms-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [JavaScript](https://www.syncfusion.com/javascript-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [Angular](https://www.syncfusion.com/angular-ui-components?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [React](https://www.syncfusion.com/react-ui-components?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [Vue](https://www.syncfusion.com/vue-ui-components?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), and [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace)), mobile ([Xamarin](https://www.syncfusion.com/xamarin-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [JavaScript](https://www.syncfusion.com/javascript-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), and [.NET MAUI](https://www.syncfusion.com/maui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace)), and desktop development ([Windows Forms](https://www.syncfusion.com/winforms-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [WPF](https://www.syncfusion.com/wpf-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [WinUI](https://www.syncfusion.com/winui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), and [.NET MAUI](https://www.syncfusion.com/maui-controls?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace)). We provide ready-to-deploy enterprise software for dashboards, reports, data integration, and big data processing. Many customers have saved millions in licensing fees by deploying our software.
```

Comment:
Recheck this section and remove UI for the platforms that we don't use it:

Reply:
Removed